rdl/data_load_tracking/DataLoadTrackerRepository.py

```diff
@@ -42,7 +42,7 @@ def complete_execution(self, execution_id, total_number_of_models,
             .one()
 
         execution_end_time = session.query(func.now()).scalar()
-        total_execution_seconds = (execution_end_time - current_execution.started_on).total_seconds()
+        total_execution_seconds = max((execution_end_time - current_execution.started_on).total_seconds(), 1)
```

Comment:
maybe add up milliseconds?

Reply:
hmm.. i don't understand.. do you mean round milliseconds? 
anyway.. the field this is saved in is for seconds.. so i'm trying to record at least 1 second and avoid recording 0 seconds in cases of close-to-zero timelines.